packages/support/lib/fs.js

```diff
@@ -1,4 +1,6 @@
 // jshint ignore: start
```

Comment:
How changes in these modules get versioned? Does CI bump their versions automatically?

Reply:
The version is determined at publish time by Lerna, based on conventional commit messages